src/main/index.js

```diff
@@ -24,7 +34,20 @@ export { downloadBlob } from './utils';
  */
 
 /**
- * @typedef {WmsLayer|XyzLayer} Layer
+ * @typedef {Object} WmtsLayer
+ * @property {'WMTS'} type
+ * @property {number} opacity Opacity, from 0 (hidden) to 1 (visible)
+ * @property {string} url A URL for the service.
```

Comment:
How about moving those to Layer type ?

Reply:
You mean a super interface layer ? I agree, I think we will need to work on that by setting up the typechecking, actually now we are writing this docjs specs but it's not interpreted anyway. We'll see how to extends interface with jsdoc annotations later on.